wrapanapi/systems/ec2.py

```diff
@@ -859,16 +858,11 @@ def get_all_unattached_volumes(self):
         return [volume for volume in self.api.get_all_volumes() if not
                 volume.attach_data.status]
 
-    def delete_sqs_queue(self, queue_name):
-        self.logger.info(" Deleting SQS queue '%s'", queue_name)
+    def delete_sqs_queue(self, queue_url):
+        self.logger.info(" Deleting SQS queue '%s'", queue_url)
```

Comment:
Can we construct this URL, instead of forcing the API change and all callers to now pass URL instead of just the name?

Reply:
I would like to do that, but apparently in boto3 queue_name == queue_url.
Just check the response for list_queues:
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sqs.html#SQS.Client.list_queues

Reply:
Same for delete_queue: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sqs.html#SQS.Client.delete_queue

Reply:
In that case I'd say keep our function's kwarg the same, and add an in-line comment here that the two are equivalent. Thoughts?

Reply:
Well, I can do that, but that function is definitely not used anywhere. I did it for the purposes of deleting SQS queue when appliance with ec2 provider added id destroyed, but never implemented.
For me it makes sense to rename it to queue_url as it's not used anywhere.